java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -405,7 +406,8 @@ long getEndStringOffset(long index) {
    */
   public final long getLong(long index) {
     // Timestamps with time values are stored as longs
-    assert type == DType.INT64 || type == DType.UINT64 || type.hasTimeResolution();
+    assert type == DType.INT64 || type == DType.UINT64 || type.hasTimeResolution()
+        || (type.isDurationType() && type != DType.DURATION_DAYS);
```

Comment:
It's getting to the point where it may be worth it to create an `EnumSet` of the types that are 64-bit integral types rather than needing to manually maintain this list in a number of different places.  Doesn't have to be done in this PR.

Reply:
I agree

Reply:
But instead of maintaining another EnumSet, I was thinking of just

```
public boolean hasSixtyFourBitRepresentation() {
   return sizeInBytes == 8;
}
```

Reply:
I take that back. That won't work because we want to restrict them to ints